test/shared/src/main/scala/zio/test/Assertion.scala

```diff
@@ -126,7 +129,9 @@ object Assertion extends AssertionVariants {
    */
   def approximatelyEquals[A: Numeric](reference: A, tolerance: A): Assertion[A] =
     Assertion[A](
-      SmartAssertions.approximatelyEquals(reference, tolerance).withCode(s"approximatelyEquals($reference, $tolerance)")
+      SmartAssertions
+        .approximatelyEquals(reference, tolerance)
+        .withCode("approximatelyEquals", param(reference), param(s"tolerance=$tolerance"))
```

Comment:
Definitely feel like you need something more here for tolerance, like the ability to name an argument.
```scala
final case class Value[a](value: a, name: Option[String] = None) extends Argument
```
perhaps

Reply:
good idea. I'll add it